app/src/organisms/Devices/InstrumentsAndModules.tsx

```diff
@@ -193,68 +187,32 @@ export function InstrumentsAndModules({
-                      : leftMountOffsetCalibration != null
-                  }
-                  mount={LEFT}
-                  robotName={robotName}
-                  is96ChannelAttached={is96ChannelAttached}
-                />
-              ) : (
-                <InstrumentCard
-                  label={t('mount', { side: 'left' })}
-                  description={t('instrument_attached')}
-                  banner={
-                    <UpdateBanner
-                      robotName={robotName}
-                      serialNumber={
-                        attachedLeftPipette != null
-                          ? attachedLeftPipette.serialNumber
-                          : 'no_left_pipette'
-                      }
-                      setShowBanner={() => null}
-                      updateType="firmware_important"
-                      handleUpdateClick={() =>
-                        setSubsystemToUpdate('pipette_left')
-                      }
-                    />
-                  }
-                />
-              )}
-              {isOT3 && badGripper == null && (
+              <PipetteCard
+                pipetteId={attachedPipettes.left?.id}
+                pipetteModelSpecs={
+                  attachedPipettes.left?.model != null
+                    ? getPipetteModelSpecs(attachedPipettes.left?.model) ?? null
+                    : null
+                }
+                isPipetteCalibrated={
+                  isOT3 && attachedLeftPipette?.ok
+                    ? attachedLeftPipette?.data?.calibratedOffset != null
+                    : leftMountOffsetCalibration != null
+                }
+                mount={LEFT}
+                robotName={robotName}
+                pipetteIs96Channel={is96ChannelAttached}
+                pipetteIsBad={badLeftPipette != null}
+                updatePipette={() => setSubsystemToUpdate('pipette_left')}
+              />
+              {isOT3 && (
                 <GripperCard
                   attachedGripper={attachedGripper}
-                  isCalibrated={attachedGripper?.data?.calibratedOffset != null}
-                />
-              )}
-              {isOT3 && badGripper != null && (
-                <InstrumentCard
-                  label={t('shared:extension_mount')}
-                  description={t('instrument_attached')}
-                  banner={
-                    <UpdateBanner
-                      robotName={robotName}
-                      serialNumber={
-                        attachedGripper != null
-                          ? attachedGripper.serialNumber
-                          : 'no_gripper'
-                      }
-                      setShowBanner={() => null}
-                      updateType="firmware"
-                      handleUpdateClick={() => setSubsystemToUpdate('gripper')}
-                    />
+                  isCalibrated={
+                    attachedGripper?.ok === true &&
```

Comment:
```suggestion
                    attachedGripper?.ok &&
```

since you do `isOT3 && attachedLeftPipette?.ok` in line 198